content/rc/api/examples/manage-subscriptions.md

```diff
@@ -11,11 +11,11 @@ aliases: /rv/api/how-to/create-and-manage-subscriptions/
          /rc/api/examples/manage-subscriptions.md
 ---
 
-You can use the Redis Enterprise Cloud REST API to create and manage a subscription.
+The Redis Enterprise Cloud REST API allows you to create and manage a subscription.
```

Comment:
I would avoid "allows you to". You could either start with the verb: 
Create and manage your subscription with the Redis Enterprise Cloud REST API. 
or use "lets" 
The Redis Enterprise Cloud REST API lets you create and manage your subscription.